doc/lua_api.txt

```diff
@@ -4938,18 +4938,22 @@ This is basically a reference to a C++ `ServerActiveObject`
     * element `stat` values:
       `position`, `name`, `scale`, `text`, `number`, `item`, `dir`
 * `hud_get(id)`: gets the HUD element definition structure of the specified ID
-* `hud_set_flags(flags)`: sets specified HUD flags to `true`/`false`
-    * `flags`: (is visible) `hotbar`, `healthbar`, `crosshair`, `wielditem`,
-      `breathbar`, `minimap`, `minimap_radar`
-    * pass a table containing a `true`/`false` value of each flag to be set or
-      unset.
-    * if a flag equals `nil`, the flag is not modified
-    * note that setting `minimap` modifies the client's permission to view the
-      minimap - the client may locally elect to not view the minimap.
-    * minimap `radar` is only usable when `minimap` is true
-* `hud_get_flags()`: returns a table containing status of hud flags
-    * returns `{hotbar=true, healthbar=true, crosshair=true, wielditem=true,
-      breathbar=true, minimap=true, minimap_radar=true}`
+* `hud_set_flags(flags)`: sets specified HUD flags of player.
+    * `flags`: A table of player HUD flags with boolean values.
+    * List of flags that can be toggled:
```

Comment:
This is still no clearer than before. This line and the previous line could be replaced with:
`flags` is a table with the following fields set to boolean values:
It's already known these are HUD flags so no need to mention that again. See nearby docs above, the word 'fields' is used instead which clarifies 'thing = value'.
```

 * `get_player_control()`: returns table with player pressed keys
* The table consists of fields with boolean value representing the pressed
```
```
 * `set_physics_override(override_table)`
* `override_table` is a table with the following fields:
```

Reply:
+1 if these 2 lines changed to:
```
`flags` is a table with the following fields set to boolean values:
```

Reply:
Isn't adding a `;` after the param name, the de facto standard? Should I squash the two lines into the following instead?
```
`flags`: A table with the following fields set to boolean values
```

Reply:
That would be fine +1.

Reply:
Done